cms/templates/js/access-editor.underscore

```diff
@@ -0,0 +1,45 @@
+<form>
+    <% if (xblockInfo.get('prereqs').length > 0) { %>
+        <h3 class="modal-section-title"><%= gettext('Limit Access') %></h3>
+        <div class="modal-section-content gating-prereq">
+            <ul class="list-fields list-input">
+                <p class="field-message">
+                    <%- gettext('Select a prerequisite subsection and enter a minimum score percentage to limit access to this subsection.') %>
+                </p>
+                <li class="field field-select">
+                    <label for="prereq" class="label">
+                        <%- gettext('Prerequisite:') %>
+                    </label>
+                    <select id="prereq" class="input">
+                        <option value="">No prerequisite</option>
+                        <% _.each(xblockInfo.get('prereqs'), function(prereq){ %>
+                            <option value="<%- prereq.block_usage_key %>"><%- prereq.block_display_name %></option>
+                        <% }); %>
+                    </select>
+                </li>
+                <li id="prereq_min_score_input" class="field field-input input-cosmetic">
+                    <label for="prereq_min_score" class="label">
+                        <%- gettext('Minimum Score:') %>
+                    </label>
+                    <input type="text" id="prereq_min_score" name="prereq_min_score" class="input input-text" size="3" aria-describedby="Minimum score must be an integer between 0 and 100." placeholder="0 - 100" />
```

Comment:
aria-describedby should be an ID name, not free-form text.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute

I suggest you write your bok choy accessibility test before fixing this bug, as it should fail (with an error that there is no element with this ID).